gedcom/__init__.py

```diff
@@ -39,14 +39,19 @@ class Gedcom:
     Elements may be accessed via:
       - a list (all elements, default order is same as in file)
       - a dict (only elements with pointers, which are the keys)
+
+    Arguments:
+
+    unicode_errors - behavior for handling unicode decode errors. Specified as
+    a value passed as the `errors` argument of open().
     """
 
-    def __init__(self, filepath):
+    def __init__(self, filepath, unicode_errors=None):
```

Comment:
Maybe it would be useful to pass the `__init__` an `options` parameter instead of just `unicode_errors`.

Also `unicode_errors` could be made less specific and more generic to something like `debug` which is a boolean, either being `True` or `False`.

Options can then be called `ParserOptions` and be stored in `self` to be later referenced in methods like `__parse`.

What do you think?